starter-code/css/main.css

```diff
@@ -0,0 +1,75 @@
+h1 a{
+	color: #ffb782;
+	letter-spacing: 2px;
+	font-weight: 500;
+	text-decoration: none;
+	font-size: 48px;
+	
+}
+
+body {
+	font-family: Lato;
+	padding: 30px 925px 0px 50px;
+}
+
+nav a{
+	font-size: 22px;
+	color: #c4c4c4;
+	padding: 0px 5px 0px 5px;
+}
+
+h2 {
+	font-weight: 500;
+	font-style: bold;
+}
+
+h3 {
+	color: #c4c4c4;
+	font-size: 35px;
+	font-weight: 500;
+	padding-top: 10px;
+}
+
+h2 a{
+	font-size: 35px;
+	text-decoration: none;
+	font-weight: 500;
+	color: black;
+}
+
+p {
+	font-size: 19px;
+}
+
+article p::first-letter { 
+    font-size: 500%;
+    color: #c4c4c4;
+    float: left;
+    padding-right: 15px;
+    line-height: 70px;
+    padding-bottom: 15px;
+
+}
+
+article {
+	width: 900px;
+	padding-top: 30px;
+	padding-bottom: 80px;
+	border-bottom: 1px solid #dbdbdb;
+}
+
+article h4 a{
+	float: right;
+	color: #ffb782;
+	font-size: 20px;
+	text-decoration: none;
+}
+
+footer nav a{
+	color: #ffb782;
+	text-decoration: none;

```

Comment:
You've started to repeat this property over and over throughout the file. Consider creating an anchor `a {  }` selector at the beginning of the file that clears this text-decoration to help keep your code DRY.